modules/cat/cat/main.nf

```diff
@@ -18,22 +18,36 @@ process CAT_CAT {
     }
 
     input:
-    path files
+    path files_in
+    val  file_out
 
     output:
-    path "file*"        , emit: file
+    path "${file_out}*" , emit: file_out
     path "*.version.txt", emit: version
 
     script:
-    def software = getSoftwareName(task.process)
-    cpus = Math.floor(task.cpus/2).toInteger()
-
-    // Use options.suffix if specified, otherwise .out; add .gz if first input file has it
-    suffix  = options.suffix ? "${options.suffix}" : ".out"
-    suffix += files[0].name =~ /\.gz/ ? '.gz' : ''
-
-    """
-    cat ${options.args} $files ${options.args2} > file${suffix}
-    cat --version | grep 'GNU coreutils' | sed 's/cat (GNU coreutils) //' > ${software}.version.txt
-    """
+    def file_list = files_in.collect { it.toString() }
+    if (file_list.size > 1) {
+
+        // | input    | output   | command1 | command2 |
+        // |----------|----------|-------- -|----------|
+        // | zipped   | zipped   | cat      |          |
+        // | unzipped | unzipped | cat      |          |
+        // | zipped   | unzipped | zcat     |          |
+        // | unzipped | zipped   | cat      | pigz     |
+
```

Comment:
This looks reasonable, suggest to replace `zipped` with `gzipped` or `compressed` and `unzipped` with `ungzipped` or `uncompressed` here and everywhere else

Reply:
Since `.gz` is what's supported (and not `.zip` or any other) I think `gzipped` and `ungzipped` are most transparent. `compressed` indicates to me any compression and `zipped` the `.zip`.

Reply:
Makes sense. Fixed https://github.com/nf-core/modules/pull/723/commits/614853fa11486ca3e2367f9fe548b25881500da4